cmd/createcluster.go

```diff
@@ -84,6 +85,7 @@ type clusterConfig struct {
 	ClusterDir string
 	Clean      bool
 	NumNodes   int
+	NumDVs     int
```

Comment:
I think we shouldn't introduce this field now when we are not having the flag for it.
Either introduce the flag and modify everything but that would make this PR big or just hardcode this with 1 at the time of creating deposit data and take care of this seperately in another PR

Reply:
yeah, let's not have the field for now then